### PR TITLE
Docs: [DS-206] [DS-230] button group & Text

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: packages/components/src/typography/Text/docs/text/nested.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Text } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <Text size="lg">
+            <Text color="primary">Nested</Text> Text
```
[the remark this answers is not in the document]
Comment (vicky-comeau) [reply packages/components/src/typography/Text/docs/text/nested.tsx:6]: no idea what that would be in this case..
Before Alex left, he did mention we should try to do things frog related..
I could put "Leap frog"
only thing that comes to mind..

[the remark this answers is not in the document]
Comment (fraincs) [reply packages/components/src/typography/Text/docs/text/nested.tsx:6]: Which can get us sued :D we did the space thing in Orbit and it wasn't that great in the end. With radioGroup and checkbox I went the real usage cases like manager, designer, etc

File: apps/docs/content/components/buttons/Button.mdx
Comment (franckgaudin) [inline apps/docs/content/components/buttons/Button.mdx:1]: It's not in the scope of your PR but could you remove the WIP from the title!

File: apps/docs/content/components/buttons/ButtonGroup.mdx [hunk 2]
```diff
@@ -1,48 +1,84 @@
...
+    ### Custom Children
+
+    TODO: Example of passing custom children to the components to fake a slot
+
+    ### Custom Component
+
+    TODO: Example of creating a custom version of this component
+</FeatureFlag>
 
 ## Migration Notes (WIP)
```
Comment (franckgaudin) [inline apps/docs/content/components/buttons/ButtonGroup.mdx:82]: You can remove the (WIP)

**Review by fraincs — APPROVED** [review]

**Pull request merged**